src/main/java/org/jenkinsci/plugins/casc/Attribute.java

```diff
@@ -95,6 +115,17 @@ public boolean isMultiple() {
         return this;
     }
 
+    public Attribute deprecated(boolean deprecated) {
+        this.deprecated = deprecated;
+        return this;
+    }
+
+    public Attribute restrictions(Class<? extends AccessRestriction>[] restrictions) {
+        this.restrictions = restrictions;
```

Comment:
You should copy the array since the objects in the array could be mutable as FindBugs complains about :sob: